pypeit/reduce.py

```diff
@@ -549,28 +551,11 @@ def find_objects_pypeline(self, image, ivar, std=False, std_trace = None, masksl
             # is. This will be a png file(s) per slit.
 
             sig_thresh = 30.0 if std else self.redux_par['sig_thresh']
-            # Parse manual
-            if self.redux_par['manual'] is not None:
-                # This requires sci_files
-                if len(sci_files) > 1:  # Can only deal with one sci file or will have to think about using the first
-                    debugger.set_trace()
-                # Generate the hand_dict
-                base_sci = os.path.basename(sci_files[0])
-                hand_extract_dict = {}
-                for mex_par in self.redux_par['manual']:
-                    if base_sci == mex_par['frame']:
-                        hand_extract_dict['hand_extract_spec'] = mex_par['spec']
-                        hand_extract_dict['hand_extract_spat'] = mex_par['spat']
-                        hand_extract_dict['hand_extract_det'] = mex_par['det']
-                        hand_extract_dict['hand_extract_fwhm'] = mex_par['fwhm']
-            else:
-                hand_extract_dict = None
-
             #
```

Comment:
Again, please make these same modifications to the echelle child, and not just the multislit child.

Reply:
done